### PR TITLE
Create docker image for running openrace

### PR DESCRIPTION
**Pull request by BradSwain**
Easy way for people to quickly try the tool without needing to install LLVM or conan.

This is a draft PR for now because it depends on:
 - [x]  #44 being merged
 - [x]  me uploading the docker image to dockerhub

### CODE REVIEW

**Comment by funemy** [comment]
#44 is merged, you can proceed with this PR.

File: README.md [hunk 1]
```diff
@@ -19,13 +19,19 @@ OpenRace is an open source implementation of the [Coderrect Scanner][CS] which i
 
 Please note, the tool is still in the early stages of development and does not support many features yet.
 
-First either download the tool from the GitHub releases page or build the latest version from source and run:
+Until our first release, the easiest way to run the tool is through the `coderrect/openrace` docker image:
 
 ```
-> openrace tests/data/integration/pthreadrace/pthreadsimple.ll
+> docker pull coderrect/openrace
+> docker run -i --rm coderrect/openrace openrace < simplepthread.ll
```
Comment (funemy) [inline README.md:26]: This looks fine to me.

But have a script to wrap the command would be better? @BradSwain what do you think?

**Review by funemy — APPROVED** [review]
All looks good.

A wrapper script for docker command would be preferable.

File: README.md (hunk 1 above)
Comment (BradSwain) [reply README.md:26]: What do you mean by a wrapper script?

Something like this?

```
#!/bin/bash
docker run -i --rm coderrect/openrace openrace < $1
```

Comment (BradSwain) [reply README.md:26]: Maybe we should add a `scripts` directory?

Comment (funemy) [reply README.md:26]: Yeah, a script like that.
Furthermore, you can add some additional logic -- like if user haven't pull the docker image, docker pull first.
We have similar stuff [here](https://github.com/coderrect-inc/coderrect/tree/develop/docker-installer), you can take that as a reference.

**Pull request merged**